kivy/core/window/__init__.py

```diff
@@ -933,6 +933,34 @@ def _set_top(self, value):
     the position set in :class:`~kivy.config.Config`.
     '''
 
+    def _get_opacity(self):
+        return self._get_window_opacity()
+
+    def _set_opacity(self, opacity):
+        self._set_window_opacity(opacity)
+        return True
```

Comment:
Always returning `True` is not a good idea as assigning the same value will always dispatch change events. For example, assigning 5 times value of `0.2` will dispatch change event 5 times but it should dispatch it only once.

I'll post suggestions on how to handle this case.